scss/_reboot.scss

```diff
@@ -26,7 +26,7 @@
 // null by default, thus nothing is generated.
 
 :root {
-  font-size: $font-size-root;
+  font-size: var(--#{$variable-prefix}#{$font-size-root});
```

Comment:
This outputs `var(--bs)`.

Should it be something like `var(--#{$variable-prefix}-root-font-size, #{$font-size-root})`, maybe?